github-electron/github-electron.d.ts

```diff
@@ -25,6 +25,25 @@ declare namespace Electron {
 		sender: EventEmitter;
 	}
 
+	type integer = number;
```

Comment:
The native bindings enforce the `integer` type, but does that necessarily impact what is enforced within TypeScript? I fear that adding an aliased `integer` type could be confusing since it wouldn't enforce the actual properties of being an `integer` anywhere. It also might make people think they need to use `Electron.integer` all over the place when they don't. I could be wrong tho.